Scuti/Scripts/UI/Store/Offer Details/UIPanningAndPinchImageLarge.cs

```diff
@@ -0,0 +1,273 @@
+﻿using UnityEngine;
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine.EventSystems;
+using UnityEngine.Events;
+using UnityEngine.UI;
+
+namespace Scuti.UI
+{
+    public class UIPanningAndPinchImageLarge : MonoBehaviour, IDragHandler, IBeginDragHandler, IEndDragHandler, IScrollHandler
```

Comment:
What is different between this large version and the other version?

Could we move the identical code out to an utility class? Or perhaps have this class extend the other and override any differences? 

We should avoid managing 2 code bases that seem very similar.

Reply:
I removed the "large" and created an abstract class for the rest, in case in the future you want to do something else in the popup part.